modules/extension/wms/src/main/java/org/geotools/data/wms/WMS1_1_1.java

```diff
@@ -16,9 +16,20 @@
  */
 package org.geotools.data.wms;
 
+import java.io.IOException;
 import java.net.URL;
+import java.util.Properties;
 
 import org.geotools.data.ows.GetCapabilitiesRequest;
+import org.geotools.data.ows.HTTPResponse;
+import org.geotools.data.ows.Response;
+import org.geotools.data.wms.request.AbstractGetStylesRequest;
+import org.geotools.data.wms.request.AbstractPutStylesRequest;
+import org.geotools.data.wms.request.GetStylesRequest;
+import org.geotools.data.wms.request.PutStylesRequest;
+import org.geotools.data.wms.response.GetStylesResponse;
+import org.geotools.data.wms.response.PutStylesResponse;
```

Comment:
@Vampouille these imports are fine, but inconsistent with the wildcards you introduced elsewhere!